### DISCUSSION TITLE
Welcome to IPL Discussions!

### DISCUSSION THREAD

**Discussion by dtrendafilov** [SofiaCPP/IPL#104] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. This will be the place, where you can:
  * Discuss the course projects and any questions about them.
  * Ask questions you’re wondering about.
  * Share and discuss ideas.

**Comment by petkobankov** [comment]
Има ли избрана дата за защита на проектите?

**Reply by HristianIliev** [reply]
+1
